packages/teams-js/src/public/interfaces.ts

```diff
@@ -577,6 +577,53 @@ export interface DialogInfo {
  */
 export type TaskInfo = DialogInfo;
 
+/**
+ *
+ * @internal
```

Comment:
We had a discussion with Lakhveer about where interfaces should go in standup on wednesday. In general, there's no strict guidance on where exported interfaces should go, what we landed on was sort of "if you think the interface is probably going to be limited to just one capability, consider putting it in that capability, while mostly interfaces.ts should be for "shared capabilities". We haven't been super careful about this in the past, so I don't think you need to move these, but it might make slightly more sense to move these into chat since they feel like they are probably only ever going to be for that capability.

Reply:
Sounds good, I'll move that now